src/main/java/com/browserstack/automate/ci/jenkins/AbstractBrowserStackReportForBuild.java

```diff
@@ -0,0 +1,36 @@
+package com.browserstack.automate.ci.jenkins;
+
+import com.browserstack.automate.ci.common.constants.Constants;
+import hudson.model.Action;
+import hudson.model.Run;
+
+public abstract class AbstractBrowserStackReportForBuild implements Action {
+    private Run<?, ?> build;
+
+    protected final String displayName = Constants.BROWSERSTACK_REPORT_DISPLAY_NAME;
+    protected final String iconFileName = Constants.BROWSERSTACK_LOGO;
+    protected final String reportUrl = Constants.BROWSERSTACK_REPORT_URL;
```

Comment:
No need of creating variables again directly use it wherever required

Reply:
done